config/global.ini.php

```diff
@@ -833,6 +833,9 @@
 ; If set to 1, when rearchiving reports in the past we do not rearchive segment data with those reports. Default is 0.
 rearchive_reports_in_past_exclude_segments = 0
 
+; Disable HTTP checks on install or Diagnostics
+disable_http_diagnostics = false
```

Comment:
@geekdenz could we use a different name for this as it only disables the private directory checks? And could we mention when someone would disable this check? 

Also  we usually use 0/1 instead of true/false.

And it might be more readable to use `enable_... = 1`.

FYI to disable external HTTP requests we already have a config: https://matomo.org/faq/troubleshooting/faq_16646/

This one won't disable all HTTP checks in diagnostics (there are few others) but only the private directory checks

Reply:
OK, thanks for letting me know. My commit below resolves this, I believe.